pom.xml

```diff
@@ -52,7 +52,7 @@
         <siddhi.map.binary.version>1.0.6</siddhi.map.binary.version>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <jacoco.version>0.7.9</jacoco.version>
-        <transport.version>6.0.74</transport.version>
+        <transport.version>6.0.69</transport.version>
```

Comment:
I think you downgraded the transport version, not the siddhi version. Can you please change the commit message ?

Reply:
Fixed

Reply:
Can you please change the commit message? Not the PR title. Because most of the users see the commit messages to get an idea about what was happened earlier in the repo. So this will confuse the users. So can you please change it? 

You can follow below steps to do it,

* Checkout the master branch of your forked repository.
* Then run "git commit --amend" command.
* Once you run the above command, you will get a chance to change the commit message.
* After that you can push the changes forcefully to the same branch of your forked repository.(using -f)

Reply:
fixed